src/rawSHA1_ng_fmt_plug.c

```diff
@@ -330,12 +330,10 @@ static void sha1_fmt_set_key(char *key, int index)
 	vtype  Z   = vsetzero();
 	vtype  X   = vloadu(key);
 	vtype  B;
-#if __AVX512F__ || __MIC__
-	uint32_t len = strlen(key);
-#else
-	uint32_t mask = vmovemask_epi8(vcmpeq_epi8(X, Z));
+
+	// FIXME: even uint64_t won't be long enough for AVX-1024
+	uint64_t mask = vcmpeq_epi8_mask(X, Z);
```

Comment:
I experimented that with raw-sha1-ng, and got some strange result.

Using `strlen`:
```
Will run 240 OpenMP threads
Benchmarking: Raw-SHA1-ng, (pwlen <= 55) [SHA1 512/512 MIC 16x]... (240xOMP) DONE
Raw:	4869K c/s real, 1408K c/s virtual
```
Using `vcmpeq_epi8_mask`:
```
Will run 240 OpenMP threads
Benchmarking: Raw-SHA1-ng, (pwlen <= 55) [SHA1 512/512 MIC 16x]... (240xOMP) DONE
Raw:	1137K c/s real, 723488 c/s virtual
```

The "real" is slower, but "virtual" gets significantly higher. 

Hints? 
(I still have a poor understanding of "real"/"virtual"...)